### PR TITLE
Japanese translation of 2024-01 blog

### PR DESCRIPTION
**Pull request by dmuelle**
post on draft: https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/ja/blog/2024/01/30/24.0.0.1.html

### CODE REVIEW

**Comment by una-tapa** [comment]
I found the leftover from the translation...  
Please remove Check out, or let me delete it. 

**Check out** [過去のOpen Liberty GAリリースのブログ記事](https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/?search=release&search!=beta)もご参照ください。(<- This means "Check out")

**Review by una-tapa — COMMENTED** [review]
Please see my comments about needing to delete "Check out"
Sorry I missed it.

File: posts/ja/2024-01-30-24.0.0.1.adoc [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+* <<instanton_features, Jakarta XML Webサービス、Java Webサービス、その他の機能をInstantOnでサポート>>
+* <<java21, Java 21用の新しいコンテナ・イメージ>>
+* <<verify_key, Open Liberty公開鍵の真正性を検証する>>
+
+link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A24001+label%3A%22release+bug%22[24.0.0.1]で修正されたバグリストをご参照ください。
+
+過去のセキュリティ脆弱性修正の一覧はlink:{url-prefix}/docs/latest/security-vulnerabilities.html[セキュリティ脆弱性（CVE）リスト]をご参照ください。
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[過去のOpen Liberty GAリリースのブログ記事]もご参照ください。
```
Comment (dmuelle) [inline posts/ja/2024-01-30-24.0.0.1.adoc]:
```suggestion
link:{url-prefix}/blog/?search=release&search!=beta[過去のOpen Liberty GAリリースのブログ記事]もご参照ください。
```

**Comment by dmuelle** [comment]
fixed

**Review by una-tapa — APPROVED** [review]
Thank you!

**Pull request merged**